src/components/Login.tsx

```diff
@@ -250,6 +252,11 @@ export default function Login({ setIdentity, lastDomain, onComplete, ...props })
                 />
               </Button>
             </Box>
+            <Box m={3} style={{ textAlign: "center"}}>
+              <Button component={Link} to="/vega-graph" variant="outlined" color="primary" style={{ width: "100%", cursor: "pointer" }}>
+                Vega Graphs
+              </Button>
+            </Box>
```

Comment:
Let's actually remove this button for now -- we don't want to confuse users but we internally can use the link since we know where it is.